### PR TITLE
Remove exit from lib code

### PR DESCRIPTION
**Pull request by jtaleric**
### Description
Instead of exit(N) from our lib we should return a empty
dict/0/false/etc.

### CODE REVIEW

File: src/touchstone/databases/elasticsearch.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def emit_compute_dict(self, uuid, compute_map, index, identifier):
                 f"Incorrect JSON data: nested dictionaries aggregations \
 fields are required in {compute_map}"
             )
-            exit(1)
+            return output_dict
```
Comment (rsevilla87) [inline src/touchstone/databases/elasticsearch.py:67]: exit(1) means there's an issue in the configuration file, why do we want to continue the execution after an issue like this?

Comment (jtaleric) [reply src/touchstone/databases/elasticsearch.py:67]: imho a lib shouldn't exit, but pass the error up to the caller. vs exit.

**Review by rsevilla87 — APPROVED** [review]
LGTM

**Pull request merged**